### ISSUE TITLE
[BUG] Cannot delete items from cart

### ISSUE THREAD

**Issue by hendi22** [Attendize/Attendize#842]
I seem to be having an issue **deleting items from the cart** once they are added.

Am I missing something or is this not possible?

Customer adds item to cart
Customer changes their mind
Customer cannot remove the item from the cart
Customer is stuck with incorrect item in cart

**Comment by johannac** [comment]
No, it's not currently possible to remove tickets from your cart after you add them. They will time out after 30 minutes. I'll add this to the features request list.
